### ISSUE TITLE
About Metric used: macro-fscore And attention visualization

### ISSUE THREAD

**Issue by Eurus-Holmes** [soujanyaporia/multimodal-sentiment-analysis#10]
Hi~ I have read your paper "Multi-level Multiple Attentions for Contextual Multimodal Sentiment Analysis", the metric used in the paper is macro-fscore, but from the source code you provide, the output is only accuracy on the test set, and the f1 value on the training set is no more than 80%. So I want to know how the macro-fscore is calculated on the test set.
And... could you tell me how to achieve attention visualization? Thanks.
